Frontend/src/components/Curso/HeaderCursos.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from "react";
```

Comment:
Create subfolders for these subcomponents and create separate styles for each one.